src/test/java/com/wikia/webdriver/testcases/mercurytests/old/curatedcontenttests/EditorTests.java

```diff
@@ -172,7 +172,7 @@ public void MercuryCuratedEditorTest_002_addAndSaveSection() {
     Assertion.assertNumber(
         curatedContent.getCuratedContentItemsNumber(),
         1,
-        "New section items"
+        "If error says that 3 elements were found - it means getList API returned cached response - ticket created: XW-1281"
```

Comment:
I would add there "probably", you never know. ;) And BTW, this is seen in the test's log, right?